library/math/numbersLine/src/components/ShowElements.tsx

```diff
@@ -76,8 +77,8 @@ const ShowElements = () => {
     });
   }, [typeRuler, windowResizing]);
 
-  return dragElements.map((element: IElement) => (
-    <div key={element.id} id={element.id} onClick={() => setIdDraggElementClick(element.id)}>
+  return dragElements.map((element: IElement, i) => (
+    <div key={element.id} id={element.id} onClick={() => setIdDraggElementClick(element.id)} style={{ zIndex: i }}>
```

Comment:
thats make you that the last element have the big zindex and its right. but what happen when i just touch element and i want him to be on the top? I